### ISSUE TITLE
Different behaviour on google chrome than firefox

### ISSUE THREAD

**Issue by frosty00** [mikeyy/nonoCAPTCHA#4]
My ip is currently banned on google chrome, and google chrome canary, and chromium but not firefox. Is there any reason for this discrepancy? I have checked the userAgent and all other request headers but I cannot find any way that google would be able to know I was using chrome over firefox. It also seems as thought I am banned on safari. Any ideas why?

https://i.imgur.com/QKZZgAF.gif

**Comment by frosty00** [comment]
EDIT: cleared my cookies on firefox and I am also banned there. May I ask if you were using proxies when you made your reddit post gif?

**Comment by mikeyy** [comment]
Yeah, I am always running with proxies when using the script. As for the ban not showing on Firefox, it might be because you were possibly currently logged into a Google account. Which will be a nice addition and is currently on my TODO list which I've been meaning to add to the repository.
